### ISSUE TITLE
[Feature Request]: ARM Template/Bicep for Synapse RBAC

### ISSUE THREAD

**Issue by georgedobrin** [Azure/ResourceModules#1972]
## Problem:
At the moment, ARM templates (and Bicep) cannot be used to grant others access to [Synapse workspaces](https://docs.microsoft.com/en-us/azure/templates/microsoft.synapse/workspaces?pivots=deployment-language-bicep) by assigning roles to users, groups, and/or service principals.
Currently, the only way to create linked services is through Synapse Analytics web dashboard or [az synapse CLI](https://docs.microsoft.com/en-us/cli/azure/synapse/role?view=azure-cli-latest)

Relevant discussions:
[Stackoverflow](https://stackoverflow.com/questions/72289140/there-is-a-way-to-set-up-synapse-administrators-with-a-bicep-module/73621178#73621178)

**Comment by cfwschmidt** [comment]
> This is likely a duplicate of #1838

I disagree that this is a duplicate. It appears #1838 doesn't mention the inability to assign the initial Synapse Workspace Admin or other Synapse workspace roles.
